SparkPost/2web2ui
# SA-1618: Active Comparisons

Note: Give your PR a short title. Something like "_TICKET NUMBER: feature description_" is good.

### What Changed
 - Bumped matchbox (4.4.0 => 4.6.0)
 - Active Metrics, Filters, Comparisons moved to collapsables
 - Active Comparisons now display
 - Comparison tags can be removed

### How To Test
 - Go to report builder and enable compare by feature flag `allow_compare_by`
 - Check that comparison, filters, and metrics are visible
 - Click `Add Comparison` and fill out compare by form with at least 2 comparisons
 - Click `Compare`
 - Verify that comparison tags are visible
 - Click remove on one of the tags
 - Verify tag is removed
 - With 2 remaining, remove a tag
 - Verify that the tag is then appended to the filters

### To Do
- [ ] Add appropriate tests
-  [ ] Check failing unit tests with matchbox upgrade
- [x] Change to default to open
